source/deployment/on-boarding.rst

```diff
@@ -67,3 +67,77 @@ Common Tasks
+
+  Consider installing and enabling the following productivity plugins for your users:   
+    - Enable audio/video calling and screen sharing with `Jitsi <https://integrations.mattermost.com/?s=Jitsi%20Plugin>`__ or `Zoom <https://integrations.mattermost.com/?s=Zoom%20Plugin>`__.
+    - Create polls with `Matterpoll <https://integrations.mattermost.com/?s=Matterpoll%20Plugin>`__.
+    - Share GIFs with `GIF Commands <https://integrations.mattermost.com/?s=gif&submit=>`__.
+    - Create and share memes with `Memes <https://integrations.mattermost.com/?s=Memes%20Plugin>`__.
+    - Set personal reminders with `Remind <https://integrations.mattermost.com/?s=Remind%20Plugin>`__.
+    - Create and share to do items with `Todo <https://github.com/mattermost/mattermost-plugin-todo>`__.
+    - Customize welcome messages for new users with `WelcomeBot <https://integrations.mattermost.com/?s=WelcomeBot%20Plugin>`__.
+
+Explore all plugins and integrations available in the `Mattermost Plugin Marketplace <https://integrations.mattermost.com/>`__.
+
+**3. Enable Automatically Extended Sessions**
+
+Keep your desktop and mobile users logged in and refresh user sessions automatically by configuring the following: **System Console > Sessions > Extend session length with activity**: ``true``. See the `Extend session length with activity <https://docs.mattermost.com/administration/config-settings.html#extend-session-length-with-activity>`__ configuration settings documentation for details. 
+
+**4. Enable Full Content Push Notifications**
+
+Enable push notifications on mobile devices to deliver messages in real time by configuring the following: **System Console > Push Notification Server > Enable Push Notifications**: ``Use TPNS``. See the `Push notification server <https://docs.mattermost.com/administration/config-settings.html#push-notification-server>`__ configuration settings documentation for details. 
+
+Enable full content push notifications, including the sender’s name, the channel name, and the message text, by configuring the following: **System Console > Notifications > Push Notification Contents**: ``Full message contents``. See the `Push notification contents <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`__ configuration settings documentation for details.
+
+.. note::
+
+  - Mattermost Enterprise E10 and E20 customers can `enable HPNS <https://docs.mattermost.com/mobile/mobile-hpns.html>`__ that includes production-level uptime SLAs.
+
+  - Mattermost Enterprise E20 customers can `enable ID-Only push notifications <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`__ so the notification message contents are not passed through Apple Push Notification Service (APNS) or Google Firebase Cloud Messaging (FCM) before reaching the device.
```

Comment:
Include link to blog post for more details on ID only notifications

Reply:
Thanks!